Controls/Controls.js

```diff
@@ -1879,23 +1881,26 @@ define([
 
 
             control.attachEventHandlers = function() {
-                $.datetimepicker.setLocale('en');
-                $("#"+control._datetimepickerid).datetimepicker({
-                    value:control._value,
-                    defaultTime:control._defaultTime,
-                    defaultDate:control._defaultDate,
-                    timepicker:control._timepicker,
-                    inline:control._inline,
-                    //mask:control._mask
-                    format:control._format,
-                    formatDate:control._formatDate,
-                    formatTime:control._formatTime,
-                    scrollMonth: control._scrollMonth,
-                    scrollTime: control._scrollTime,
-                    scrollInput: control._scrollInput
-
+                Promise.all([
+                    import("jquery-datetimepicker"),
+                    import("jquery-datetimepicker/jquery.datetimepicker.css")
```

Comment:
This is a feature of Webpack called [Dynamic Imports](https://webpack.js.org/guides/code-splitting/#dynamic-imports). This specific library only needs to be loaded when a date selection control is used, so the bytes are only transmitted to the browser on first use. Since it's no longer loaded at bundle-time but at run-time, this operation becomes asynchronous which is why `import()` returns a promise.

It was mostly an experiment and in the grander scheme of things I think the added complexity probably isn't worth it.